initial-setup.spec

```diff
@@ -43,15 +43,39 @@ a series of steps that allows for easier configuration of the machine.
 Summary: Graphical user interface for the initial-setup utility
 Requires: gtk3
 Requires: anaconda-gui >= %{anacondaver}
-Requires: firstboot(windowmanager)
-Requires: xorg-x11-xinit
-Requires: xorg-x11-server-Xorg
+Requires: firstboot(gui-backend)
 Requires: %{name} = %{version}-%{release}
+Suggests: %{name}-gui-wayland-generic
 
 %description gui
 The initial-setup-gui package contains a graphical user interface for the
 initial-setup utility.
 
+%package gui-xorg
+Summary: Run the initial-setup GUI in Xorg
+Requires: xorg-x11-xinit
```

Comment:
This should probably have `Requires: %{name}-gui = %{version}-%{release}` also, just so people can just pick this package and get the right stuff.